src/css/top-section.css

```diff
@@ -0,0 +1,307 @@
+.mobile-menu {
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  padding: 1rem 2rem;
+  color: var(--black);
+  font-size: 2.5rem;
+}
+
+.menu-mobile {
+  display: flex;
+  flex-direction: column;
+  height: max-content;
+  background: rgba(89, 89, 89, 95%);
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 57px;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  z-index: 999;
+  animation: slide-in 1s both;
+}
+
+.hideMenu {
+  display: none;
+}
+
+.buy-ticket {
+  border: 4px solid var(--orange);
+  padding: 0.5rem 2rem;
+  color: var(--orange);
+  font-weight: 700;
+  cursor: pointer;
+}
+
+.close {
+  color: #e1d7d4;
+  align-self: end;
+  margin: 3.875rem 2.375rem -0.875rem;
+  background-color: unset;
+  width: 1rem;
+  height: 1rem;
+  z-index: 989;
+  border: none;
+}
+
+.close:hover {
+  background-color: unset;
+  color: #6f6c6b;
+  border: none;
+  transform: rotate(180deg);
+  scale: 1.5;
+}
+
+.mobileLinkText {
+  text-decoration: none;
+  color: #fff5e1;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+}
```

Comment:
- [ ] I like that you're trying to add your own flavor 🧂 here but it's best to stay consistent in your design and use the given font from the design (Cocogoose, Lato). 
- [ ] You can just remove the `font-family` property at line `59`